sphinx-doc/source/installation.rst

```diff
@@ -20,8 +20,8 @@ Introduction
 
 The most fundamental dependencies of CATMAID are:
 
-1. PostgreSQL >= 11 and PostGIS >= 2.5 (recommended: PostgreSQL 12 + PostGIS 3)
-2. Python 3.5, 3.7 or PyPy3.6
+1. PostgreSQL 12 and PostGIS 2.5
+2. Python 3.6, 3.7 or PyPy3.6
```

Comment:
Do we support 3.8? The travis build passes, at least.

Reply:
Good question, I think we actually do. :-) I just replaced my local `virtualenv` with a 3.8 version, let's see how that goes. All possible dependencies seem to install without problems (just pandas took ages to compile things). This is probably unrelated to this PR though. I just fixed the 3.5 reference above, because don't support that anymore.

Reply:
BTW, I noticed that with Python 3.8 the devserver starts noticeably faster than with 3.6. That's nice and suggests that other things might be faster too, maybe we should starting using Python 3.8 as default.